zio-http/shared/src/main/scala/zio/http/codec/HttpContentCodec.scala

```diff
@@ -216,7 +216,7 @@ object HttpContentCodec {
       ZPipeline.identity[Byte]
   }
 
-  val byteCodec: HttpContentCodec[Byte] = {
+  implicit val byteCodec: HttpContentCodec[Byte] = {
     HttpContentCodec(
       ListMap(
         MediaType.allMediaTypes.filter(_.binary).map(mt => mt -> ByteBinaryCodec): _*,
```

Comment:
Same as the above

Reply:
@987Nabil FYI: I remember you were skeptical of having the primitive codecs implicit. Please consider these changes are merged as they are when you redesign the codecs.